.travis.yml

```diff
@@ -134,43 +131,21 @@ matrix:
     - env:
         - NAME=astyle
       install:
-      - curl -L0 https://mbed-os.s3-eu-west-1.amazonaws.com/builds/deps/astyle_3.1_linux.tar.gz --output astyle.tar.gz;
-        mkdir -p BUILD && tar xf astyle.tar.gz -C BUILD;
-        pushd BUILD/astyle/build/gcc;
-        make;
-        export PATH=$PWD/bin:$PATH;
-        popd;
-      - astyle --version
+        - >-
+          curl -L0 https://mbed-os.s3-eu-west-1.amazonaws.com/builds/deps/astyle_3.1_linux.tar.gz --output astyle.tar.gz;
+          mkdir -p BUILD && tar xf astyle.tar.gz -C BUILD;
+          cd BUILD/astyle/build/gcc;
+          make;
+          export PATH=$PWD/bin:$PATH;
+          cd -
+        - astyle --version
       script:
-        # only changed files this time
-        git diff --name-only --diff-filter=d $TRAVIS_BRANCH | grep '.*\.\(h\|c\|hpp\|cpp\)$' | fgrep -v -f .astyleignore | xargs -n 100 -I {} bash -c "astyle -n --options=.astylerc \"{}\"" > astyle-files-changed.out;
-          if [ $(cat astyle-files-changed.out | grep Formatted | wc -l) -ne 0 ]; then
-            git --no-pager diff;
-            echo "";
-            echo "AStyle check failed, please fix style issues as shown above";
-            (exit 1);
-          else
-            echo "Coding style check OK";
-          fi
-      after_success:
-        # run astyle for all files on the branch
-        - git checkout -- .
-        - find -regex '.*\.\(h\|c\|hpp\|cpp\)$' -type f | fgrep -v -f .astyleignore | xargs -n 100 -I {} bash -c "astyle -n --options=.astylerc \"{}\"" > astyle-branch.out;
-        # update status if we succeeded, compare with master if possible
-        - |
-          CURR=$(cat astyle-branch.out | grep Formatted | wc -l)
-          PREV=$(curl -u "$MBED_BOT" https://api.github.com/repos/$TRAVIS_REPO_SLUG/status/master \
-              | jq -re "select(.sha != \"$TRAVIS_COMMIT\")
-                  | .statuses[] | select(.context == \"travis-ci/$NAME\").description
-                  | capture(\", (?<files>[0-9]+) files\").files" \
-              || echo 0)
+        - >-
+          git diff --name-only HEAD..${TRAVIS_BRANCH} \
```

Comment:
--diff-filter=d should be kept - we should not check removed files

Reply:
Counterpoint.

If a file was removed, doesn't that mean/imply that it passed astyle to begin with and wouldn't be flagged anyways?